azure-pipelines.yml

```diff
@@ -58,9 +59,10 @@ jobs:
       meson --version
       doxygen --version
       echo "PATH=$PATH"
-    displayName: Display tool versions
+      printenv
+    displayName: Display environment information
   - bash: |
-      fork_origin="$(git merge-base --fork-point origin/master)"
+      fork_origin="$(git merge-base --fork-point origin/$SYSTEM_PULLREQUEST_TARGETBRANCH)"
```

Comment:
We should really pull the `fork_origin` and the `changed_files` out into a separate file, but later.